docs/UserGuide.md

```diff
@@ -63,10 +63,19 @@ Examples:
 Lists all currently supports commands, and provides a link to this user guide.<br>
 Format: `help`
 
-### Listing all anime: `list`
+### Listing anime: `list`
 
 Shows a list of all anime names the user has added.<br>
-Format: `list`
+An additional status parameter can be specified to show only anime with that status.<br>
+Format: `list [s/STATUS]`
+
+- `STATUS` refers to the updated status of the anime.
+- Available statuses: "towatch", "watching", "finished"
```

Comment:
```suggestion
- Available statuses: `towatch`, `watching`, `finished`
```

Reply:
im changing them to use code block in my end, so should change here to make it consistent